graphs/rdf-insert.fbp

```diff
@@ -0,0 +1,27 @@
+# rdf-insert.fbp
+
+# Takes parsed JSON-LD and inserts it into the target-graph (IRI) at the sparql-endpoint.
```

Comment:
Based on the discussion this morning, I thought there was going to be a bit more documentation here?